Don't include our own declaration of setpgrp if the system defines prototypes
for SysV functions (like setpgrp()).  Not sure if this is the Right fix or not.
--- tcsh-6.12.00/sh.h	2004-02-10 11:35:44.000000000 -0500
+++ tcsh-6.12.00/sh.h	2004-02-10 11:35:40.000000000 -0500
@@ -474,7 +474,9 @@
 extern pid_t getpgrp __P((void));
 #  endif	/* BSD || SUNOS4 || IRISD || DGUX */
 # endif /* POSIX */
+#ifndef _SVID_SOURCE
 extern pid_t setpgrp __P((pid_t, pid_t));
+#endif
 #endif /* !WINNT_NATIVE */
 
 typedef sigret_t (*signalfun_t) __P((int));
